PaloAltoCortexXDR/action_scan_endpoints.json

```diff
@@ -0,0 +1,83 @@
+{
+  "arguments": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
```

Comment:
From [the documentation](https://cortex-panw.stoplight.io/docs/cortex-xdr/mh0ogcjyxxbek-scan-endpoints), it seems that we can filter according to a list of usernames.
Why this filter was not retained?